app/assets/stylesheets/modules/forms.css.scss

```diff
@@ -100,16 +100,28 @@
 }
 
 
-// .form-control {
-// border: 2px solid #bdc3c7;
-// color: #34495e;
-// line-height: 1.467;
-// padding: 8px 12px;
-// height: 42px;
-// -webkit-appearance: none;
-// border-radius: 6px;
-// -webkit-box-shadow: none;
-// box-shadow: none;
-// -webkit-transition: border .25s linear, color .25s linear, background-color .25s linear;
-// transition: border .25s linear, color .25s linear, background-color .25s linear;
-// }
\ No newline at end of file
+ .form-control {
```

Comment:
Changing this appears to affect many, many views (.html pages) -- can you confirm that you tested all of them to ensure their behavior and appearance remains as intended?